packages/web-components/src/components/masthead/megamenu-top-nav-menu.ts

```diff
@@ -102,17 +102,19 @@ class DDSMegaMenuTopNavMenu extends DDSTopNavMenu {
         ?.shadowRoot?.querySelector('.bx--masthead__l0');
 
       if (this.expanded) {
-        doc?.body?.classList.add(`${prefix}--body__lock-scroll`);
         doc.body.style.marginRight = `${this._scrollBarWidth}px`;
+        doc.body.style.position = `relative`;
+        doc.body.style.overflow = `hidden`;
```

Comment:
one more thing, could you also remove the `@import '@carbon/ibmdotcom-styles/scss/components/masthead/masthead-megamenu-scroll';` import from `packages/web-components/.storybook/container.scss`? Since it's not necessary anymore, thanks!